app/controllers/erdbeere_controller.rb

```diff
@@ -17,7 +17,7 @@ def show_example
   end
 
   def show_property
-    response = Faraday.get(ENV.fetch("ERDBEERE_API") + "/properties/#{params[:id]}")
+    response = Clients::ErdbeereClient.get("properties/#{params[:id]}")
```

Comment:
If an endpoint cannot be reached (e.g. locally kill the erdbeere docker container), we get an error on this page: http://localhost:3000/lectures/1/show_structures

In this case, we could show a nice error message to the user saying that the erdbeere server that hosts the examples is unfortunately not reachable at the moment, otherwise we might be spammed with these errors in our email box and it's not the best experience for the user neither. In addition, we could be informed by email via the erdbeere service itself that it is down.

Or we might reserve this better error handling for later and just get erdbeere up and running for now ;)

Reply:
I've verified that error handling is graceful in case of editing a lecture via http://localhost:3000/lectures/1/edit#info

Reply:
I added better error handling in the latest commit. Please check if it works.